src/datasources/data-frame/types.ts

```diff
@@ -26,7 +26,7 @@ export const defaultQuery: Omit<ValidDataFrameQuery, 'refId'> = {
 };
 
 export const DataFrameFeatureTogglesDefaults: DataFrameFeatureToggles = {
-  queryByDataTableProperties: false,
+  queryByDataTableProperties: true,
```

Comment:
[Do not enable the toggle by default](https://dev.azure.com/ni/DevCentral/_git/Skyline/pullRequest/1028288#1757367524) until [Feature 3161151: Metadata Query API Improvements](https://dev.azure.com/ni/DevCentral/_workitems/edit/3161151/) is complete